service/internal/presigned-url/middleware_test.go

```diff
@@ -0,0 +1,14 @@
+package presignedurl
+
+import "testing"
+
+func TestPresignMiddleware(t *testing.T) {
```

Comment:
Probably want a GitHub issue to track this.

Reply:
Thanks! that was completely missed. I'll fix this up...